watertap/costing/units/gac.py

```diff
@@ -19,106 +21,162 @@
     )
-
     blk.adsorbent_unit_cost_exp_coeff = pyo.Var(
         initialize=-1.25311e-5,
         units=pyo.units.kg**-1,
         doc="GAC adsorbent exponential cost parameter coefficient",
     )
-
-    blk.other_cost_coeff = pyo.Var(
+    blk.pres_other_cost_coeff = pyo.Var(
         initialize=16660.7,
         units=pyo.units.USD_2020,
-        doc="GAC other cost power law coefficient",
+        doc="GAC pressure other cost power law coefficient",
     )
-
-    blk.other_cost_exp = pyo.Var(
+    blk.pres_other_cost_exp = pyo.Var(
         initialize=0.552207,
         units=pyo.units.dimensionless,
-        doc="GAC other cost power law exponent",
+        doc="GAC pressure other cost power law exponent",
+    )
+    blk.grav_other_cost_coeff = pyo.Var(
+        initialize=38846.9,
+        units=pyo.units.USD_2020,
+        doc="GAC gravity other cost power law coefficient",
+    )
+    blk.grav_other_cost_exp = pyo.Var(
+        initialize=0.490571,
+        units=pyo.units.dimensionless,
+        doc="GAC gravity other cost power law exponent",
     )
-
     blk.regen_frac = pyo.Var(
         initialize=0.70,
         units=pyo.units.dimensionless,
         doc="Fraction of spent GAC adsorbent that can be regenerated for reuse",
     )
-
     blk.regen_unit_cost = pyo.Var(
         initialize=4.28352,
         units=pyo.units.USD_2020 * pyo.units.kg**-1,
         doc="Unit cost to regenerate spent GAC adsorbent by an offsite regeneration facility",
     )
-
     blk.makeup_unit_cost = pyo.Var(
         initialize=4.58223,
         units=pyo.units.USD_2020 * pyo.units.kg**-1,
         doc="Unit cost to makeup spent GAC adsorbent with fresh adsorbent",
     )
+    blk.pres_energy_coeff_0 = pyo.Var(
+        initialize=8.09926e-4,
+        units=pyo.units.kW,
+        doc="GAC steel pressure contactor polynomial energy consumption coefficient 0",
+    )
+    blk.pres_energy_coeff_1 = pyo.Var(
+        initialize=8.70577e-4,
+        units=pyo.units.kW * (pyo.units.m**3) ** -1,
+        doc="GAC steel pressure contactor polynomial energy consumption coefficient 1",
+    )
+    blk.pres_energy_coeff_2 = pyo.Var(
+        initialize=0,
+        units=pyo.units.kW * (pyo.units.m**3) ** -2,
+        doc="GAC steel pressure contactor polynomial energy consumption coefficient 2",
+    )
+    blk.grav_energy_coeff_0 = pyo.Var(
+        initialize=0.123782,
+        units=pyo.units.kW,
+        doc="GAC concrete gravity contactor polynomial energy consumption coefficient 0",
+    )
+    blk.grav_energy_coeff_1 = pyo.Var(
+        initialize=0.132403,
+        units=pyo.units.kW * (pyo.units.m**3) ** -1,
+        doc="GAC concrete gravity contactor polynomial energy consumption coefficient 1",
+    )
+    blk.grav_energy_coeff_2 = pyo.Var(
+        initialize=-1.41512e-5,
+        units=pyo.units.kW * (pyo.units.m**3) ** -2,
+        doc="GAC concrete gravity contactor polynomial energy consumption coefficient 2",
+    )
 
```

Comment:
Are we fixing all of these coefficients somewhere? Also, it looks like regardless of the contactor type specified, all of the cost variables will be created for both pressure and gravity-fed contactors. Is that right?